demo/server/src/modules/keycloak/index.ts

```diff
@@ -0,0 +1,36 @@
+/// <reference path="./keycloak.d.ts" />
+
+import * as express from 'express';
+import * as session from 'express-session';
+import Keycloak = require('keycloak-connect');
+
+// Express Session Configuration.
+const sessionOpts = {
```

Comment:
Good idea. Json files are hard to deal with in typescript so best to use ts/js file instead.